bindata/etcd/cluster-backup.sh

```diff
@@ -110,6 +112,10 @@ if [ ! -f "$ETCDCTL_CACERT" ] && [ ! -d "${CONFIG_FILE_DIR}/static-pod-certs" ];
 fi
 
 dl_etcdctl
-backup_latest_kube_static_resources "${BACKUP_RESOURCE_LIST[@]}"
+backup_latest_kube_static_resources "${BACKUP_TAR_FILE}" "${BACKUP_RESOURCE_LIST[@]}"
```

Comment:
if the list is only used to pass to the function why not just use the list in the function?

https://github.com/openshift/cluster-etcd-operator/pull/603/files#diff-b15139535382251f9a3e1ca944f23d804ad410e7a78107940e5f390b64487113R102

Reply:
To maintain the symmetry with the function restore_static_pods() in `cluster-restore.sh`.

But I can change it.

Reply:
Pushed the change.